app/controllers/Shipping.scala

```diff
@@ -1,12 +1,14 @@
 package controllers
-import actions.CommonActions._
+import javax.inject.Inject
+
+import actions.CommonActions
 import com.gu.i18n.CountryGroup.UK
 import com.gu.memsub.subsv2.CatalogPlan
 import model.Subscriptions._
 import play.api.mvc._
 import services.TouchpointBackend
 
-object Shipping extends Controller {
+class Shipping @Inject()  extends Controller with CommonActions {
```

Comment:
If you're not injecting anything, this could remain an object.

Reply:
I think it's probably worth converting all of the controllers to classes so that a) you have consistency and b) you don't need to use the StaticRoutesGenerator.